projects/plugins/protect/src/js/routes/scan/index.jsx

```diff
@@ -39,7 +39,11 @@ const ScanPage = () => {
 	}
 
 	const hasHistory = history && history.threats.length;
+<<<<<<< HEAD
 	const showResults = !! status.threats.length || hasHistory;
+=======
+	const showResults = status.threats.length || hasHistory;
+>>>>>>> 1850b0c9a6 (Protect: only show Threats DataViews when scan is in progress or has results to show)
 
```

Comment:
Maybe:
```
const hasActiveThreats = status && status.threats.length;
const hasHistory = history && history.threats.length;
const showResults = hasActiveThreats || hasHistory;
```

Reply:
Thanks for catching this! Fixed with https://github.com/Automattic/jetpack/pull/40621#discussion_r1889373979